edenai_apis/utils/constraints.py

```
@@ -247,10 +247,15 @@ def validate_models(
     if "text_to_speech" in subfeature and voice_ids:
         if any(option in voice_ids for option in ["MALE", "FEMALE"]):
-            voice_id = retreive_voice_id(
-                provider, subfeature, args["language"], args["option"], settings
-            )
-            args["voice_id"] = voice_id
+            try:
+                voice_id = retreive_voice_id(
+                    provider, subfeature, args["language"], args["option"], settings
+                )
+                args["voice_id"] = voice_id
+            except Exception as exc:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)